package.json

```diff
@@ -63,9 +63,8 @@
     "element-collapse": "1.1.0",
     "esbuild": "0.20.1",
     "eslint": "8.57.0",
-    "@warp-ds/eslint-config": "1.0.4",
-    "eslint-plugin-prettier": "5.1.3",
-    "eslint-config-prettier": "9.1.0",
```

Comment:
@imprashast, although all seems to work fine, I'd rather double check with you - if these are defined as `peerDependencies` in @warp-ds/esling-config, does it mean we DO or DO NOT need to install those wherever the package is used?

Reply:
@felicia-haggqvist already did talk to me about these. If you have peerDeps in 1 package, then you can skip it from the one that's using that package :)

Reply:
I've been trying to google around before and my impression was that `peerDependencies` are used when the user of a package like `@warp-ds/eslint-config` is supposed to provide the dependent package. For example this is what I read in this article about peerDependencies: https://dev.to/adamgoth/understanding-package-dependencies-within-a-pnpm-monorepo-19ge

> When you specify a package as a peerDependency in your package.json file, you are essentially saying that "this > package requires that the consumer of my package provides this package".

If I'm misunderstanding some concepts of pnpm/npm/whatever, I would very much appreciate some guidance 😊